src/Features/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -180,7 +180,6 @@ Microsoft.CodeAnalysis.Completion.CompletionRules.WithDismissIfEmpty(bool dismis
 Microsoft.CodeAnalysis.Completion.CompletionRules.WithDismissIfLastCharacterDeleted(bool dismissIfLastCharacterDeleted) -> Microsoft.CodeAnalysis.Completion.CompletionRules
 Microsoft.CodeAnalysis.Completion.CompletionRules.WithSnippetsRule(Microsoft.CodeAnalysis.Completion.SnippetsRule snippetsRule) -> Microsoft.CodeAnalysis.Completion.CompletionRules
 Microsoft.CodeAnalysis.Completion.CompletionService
-Microsoft.CodeAnalysis.Completion.CompletionService.CompletionService() -> void
```

Comment:
@jasonmalinowski Do we have a list of breaking changes somewhere that this should be added to?

Reply:
https://github.com/dotnet/roslyn/blob/main/docs/Breaking%20API%20Changes.md

Reply:
Added a note there.